packages/expo-av/src/AV.ts

```diff
@@ -10,6 +10,12 @@ import { Platform } from 'expo-core';
 //  API to select stream type on Android
 //  subtitles API
 
+export enum PitchCorrectionQuality {
+  Low = ExponentAV.Qualities && ExponentAV.Qualities.Low,
```

Comment:
actually it should be 
```suggestion
  Low = ExponentAV && ExponentAV.Qualities && ExponentAV.Qualities.Low,
```
that's why I dislike this native-constants-to-enum way 😅